src/main/java/com/wrapper/Controller/HRP/HipGateway.java

```diff
@@ -0,0 +1,81 @@
+package com.wrapper.Controller.HRP;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.wrapper.Properties.ApplicationConstantConfig;
+import com.wrapper.ResponseController.LinkRecordsResponse;
+import com.wrapper.ResponseController.OnConfirmResponse;
+import com.wrapper.ResponseController.OnInitResponse;
+import com.wrapper.ServiceImpl.CareContextTableService;
+import com.wrapper.Services.AddCareContext;
+import com.wrapper.Services.CommonServices;
+import com.wrapper.Services.ConfirmAuth;
+import com.wrapper.Services.Link_init;
+import com.wrapper.Utility.CommonUtility;
+import java.io.FileNotFoundException;
+import java.net.URISyntaxException;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.client.RestTemplate;
+
+@RestController
+public class HipGateway {
```

Comment:
I dont think this is a controller. 

Probably another service layer that the HIP Facade uses.